mono/utils/mono-tls.c

```diff
@@ -31,7 +31,7 @@
 #define MONO_THREAD_VAR_OFFSET(var,offset) __asm ("movl $" #var "@ntpoff, %0" : "=r" (offset))
 #endif
 
-#elif defined(TARGET_ARM64) && !defined(PIC)
+#elif defined(TARGET_ARM64) && !defined(PIC) && !defined(HOST_WIN32)
```

Comment:
Doesn't need addressed in this PR, but I'd like to understand state of TLS for current mono platforms. There are a few arch checks later in the file, and it's unclear to me what windows on amd64/x86 is using and how ARM64 will compare to that.